docker-compose.yml

```diff
@@ -30,3 +30,14 @@ services:
       - django
     ports:
       - "80:80"
+
+  render:
+    build:
```

Comment:
I removed it (since it isn't necessary) because I was having issues with my build earlier where it was actually using the wrong docker file (Django) even though my context was within `./render`.